chef/cookbooks/crowbar/recipes/crowbar-upgrade.rb

```diff
@@ -102,6 +107,7 @@
           crm resource stop $resource
         done
       done
+      cibadmin -E -f
```

Comment:
Only for readability: I'd use an `execute` chef resource like what you did above. This makes it easier to compare the two code paths.